salt/worker/manager.sls

```diff
@@ -0,0 +1,30 @@
+#testswarm:
+#  cmd.run:
+#    - name: docker swarm join-token worker -q
+
+
+/var/worker/manager.sh:
+  cmd.run:
+    - name: docker swarm init  --advertise-addr eth0
```

Comment:
eth0 is just to generate a command line  for easy use and no set the listen address